pkg/kubelet/kubelet.go

```diff
@@ -2003,6 +2119,8 @@ func (kl *Kubelet) HandlePodCleanups() error {
 		glog.Errorf("Failed cleaning up bandwidth limits: %v", err)
 	}
 
+	kl.cleanupOrphanedPodCgroups(allPods, runningPods)
```

Comment:
should this call only be called if the feature is enabled, or should the call-ed function work with or without the feature on?